### PR TITLE
Migration/strokes

### PR DESCRIPTION
**Pull request by rvilarl**
Strokes!

### CODE REVIEW

**Comment by 0xfe** [comment]
Hi @rvilarl -- apologies for the slow reviews the past few days, I got sucked into some work stuff that's taking all my time. Will address these as soon as I can. Thanks for keeping them going!

**Comment by rvilarl** [comment]
I am wishing to finish. I started that and I will not leave it in the middle. It is a lot of effort😀

File: src/strokes.ts [hunk 1]
```diff
@@ -9,39 +9,64 @@ import { Vex } from './vex';
 import { Modifier } from './modifier';
 import { StaveNote } from './stavenote';
 import { Glyph } from './glyph';
+import { Note } from './note';
+import { FontInfo } from './types/common';
+import { TabNote } from './tabnote';
+import { ModifierContextState } from './modifiercontext';
+
+export interface StrokeOptions {
```
Comment (0xfe) [inline src/strokes.ts:17]: This doesn't seem to be used anywhere.

File: src/strokes.ts [hunk 2]
```diff
@@ -9,39 +9,64 @@ import { Vex } from './vex';
...
+  all_voices: boolean;
+}
+
+export interface StrokeRenderOptions {
```
Comment (0xfe) [inline src/strokes.ts:21]: This can be inline.

File: src/strokes.ts (hunk 1 above)
Comment (rvilarl) [reply src/strokes.ts:17]: removed

File: src/strokes.ts (hunk 2 above)
Comment (rvilarl) [reply src/strokes.ts:21]: done

**Comment by rvilarl** [comment]
Ready to merge

**Comment by 0xfe** [comment]
LGTM, thanks!

**Pull request merged**